core/src/main/scala/akka/kafka/ConsumerSettings.scala

```diff
@@ -199,7 +206,8 @@ class ConsumerSettings[K, V] @InternalApi private[kafka] (
     val offsetForTimesTimeout: FiniteDuration,
     val metadataRequestTimeout: FiniteDuration,
     val drainingCheckInterval: FiniteDuration,
-    val consumerFactory: ConsumerSettings[K, V] => Consumer[K, V]
+    val consumerFactory: ConsumerSettings[K, V] => Consumer[K, V],
+    val kafkaConnectionCheckerSettings: Option[KafkaConnectionCheckerSettings]
```

Comment:
Instead of the none option, use an "empty" implementation which doesn't do the checking.

Reply:
You mean some kind of config with single value enable:false?

Reply:
Yes, add enabled to the settings and test it to decide if the actor is started. The default value is `false` already.